config/jobs/kubernetes/sig-node/crio.yaml

```diff
@@ -1,6 +1,7 @@
 periodics:
 - name: ci-crio-cgroupv1-node-e2e-conformance
   interval: 1h
+  decorate: true
```

Comment:
Scanning thru the document quickly I see at least these two needs to be made for THIS job:

- add extra refs for kubernetes repo

```
 extra_refs:
  - org: kubernetes
    repo: kubernetes
    base_ref: master
    path_alias: k8s.io/kubernetes
```

- drop --root,  and --scenario

Reply:
other jobs may need more changes. I would recommend to do it one job at a time

Reply:
sounds good.  I'll do this for the e2e eviction tests and we can see.

Reply:
So I am thinking I'll iterate on https://github.com/kubernetes/test-infra/pull/30720 and get that converted.  And then we can revisit this PR again.